PRIVACY.md

```diff
@@ -3,27 +3,48 @@ layout: layouts/privacy.html
 permalink: /privacy/index.html
 ---
 
-# SimpleX Chat Privacy Policy and Conditions of Use
+# SimpleX Chat Operators Privacy Policy and Conditions of Use
 
-SimpleX Chat is the first communication network based on a new protocol stack that builds on the same ideas of complete openness and decentralization as email and web, with the focus on providing security and privacy of communications, and without compromising on usability.
+## Summary
 
-SimpleX Chat communication protocol is the first protocol that has no user profile IDs of any kind, not even random numbers, cryptographic keys or hashes that identify the users. SimpleX Chat apps allow their users to send messages and files via relay server infrastructure. Relay server owners and providers do not have any access to your messages, thanks to double-ratchet end-to-end encryption algorithm (also known as Signal algorithm - do not confuse with Signal protocols or platform) and additional encryption layers, and they also have no access to your profile and contacts - as they do not provide any user accounts.
+[Introduction](#introduction) and [General principles](#general-principles) cover SimpleX Chat network design, the network operators, and the general principles for privacy and security that SimpleX network provides.
+
+[Privacy policy](#privacy-policy) covers:
+- data stored only on your device - [your profiles](#user-profiles), delivered [messages and files](#messages-and-files). This information can be transferred to another device, and you are responsible for its preservation - if you delete the app it will be lost.
+- [private message delivery](#private-message-delivery) that protects your IP address and contacts from the destination servers.
+- information is stored on the servers ([undelivered messages and files](#storage-of-messages-and-files-on-the-servers)). It also explains how [connections between users](#connections-with-other-users) work.
+- [iOS push notifications](#ios-push-notifications) privacy limitations.
+- [User support](#user-support), [SimpleX directory](#simplex-directory) and [any other data](#another-information-stored-on-the-servers) that may be stored on the servers.
+- [Preset server operators](#preset-server-operators) and the [information they can share](#information-preset-operators-may-share)
+- [Source code license](#source-code-license) and [updates to this document](#updates).
+
+[Conditions of Use of Software and Infrastructure](#conditions-of-use-of-software-and-infrastructure) cover the conditions you need to accept to use SimpleX Chat Applications and the relay servers of preset operators. Their purpose is to protect preset server operators and other users from any malicious actions of other users.
+
+This summary is provided for convenience only, it is not part of the Privacy Policy and Conditions of Use.
```

Comment:
probably better to use same name everywhere? or give short names in previous paragraph: `Conditions of Use of Software and Infrastructure (further, Conditions of Use [or Conditions]) `